### PR TITLE
View index job

### PR DESCRIPTION
**Pull request by OmarBerrada**

### CODE REVIEW

File: app/views/jobs/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+<p><%= job.description %></p>
+<p><%= job.budget %></p>
+<% end %>
+
+</br>
+</br>
+
+<!-- Add a new job -->
+
+<%= link_to "Add new job", new_cocktail_path %>
```
Comment (contatc2) [inline app/views/jobs/index.html.erb]: not sure the path is correct

File: app/views/jobs/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+
+</br>
+</br>
+
+<!-- Go back to your profile -->
+
+<%= link_to "Back to my profile", user_path(@user) %>
```
Comment (contatc2) [inline app/views/jobs/index.html.erb]: does this page exist?

File: app/views/jobs/index.html.erb [hunk 3]
```diff
@@ -1,2 +1,26 @@
-<h1>Jobs#index</h1>
+<!-- <h1>Jobs#index</h1>
 <p>Find me in app/views/jobs/index.html.erb</p>
+ -->
+
+<h1>List of jobs</h1>
+
+<% @jobs.each do |job| %>
+<li> <%= link_to "#{job.name}", jobs_show(job) %> </li>
```
[the remark this answers is not in the document]
Comment (OmarBerrada) [reply app/views/jobs/index.html.erb]: ok i do it

File: app/views/jobs/index.html.erb (hunk 1 above)
Comment (OmarBerrada) [reply app/views/jobs/index.html.erb]: ok we use new_job_path instead

File: app/views/jobs/index.html.erb (hunk 2 above)
Comment (OmarBerrada) [reply app/views/jobs/index.html.erb]: should we use this instead: edit_user_registration_path(@user) ?

**Pull request merged**